tests/karma/unit/controllers/contacts-content.js

```diff
@@ -221,5 +236,21 @@ describe('ContactsContentCtrl', function() {
       });
     });
 
+    it('includes subjectIds in reports search so JSON reports are found', function(done) {
+      childContactPerson.patient_id = 'cd';
+      childContactPerson.place_id = 'ef';
+      stubSearch(null, [ { _id: 'ab' } ]);
+      runPersonTest(done, doc, null, function(selected) {
+        chai.expect(selected.reports.length).to.equal(1);
+        chai.expect(selected.reports[0]._id).to.equal('ab');
```

Comment:
can you move that test of `selected.reports` to a separate test? (Like 'it displays the reports returned by the search' or something)